docs/usage.md

```diff
@@ -408,275 +589,267 @@ The syntax for this reference configuration is as follows:
+      dbsnp                    = '<path to the dbsnp file>'
+      dbsnp_index              = '<path to the dbsnp index>'
+      dict                     = '<path to the dict file>'
+      fasta                    = '<path to the fasta file>'
+      fasta_fai                = '<path to the fasta index>'
+      germline_resource        = '<path to the germline_resource file>'
+      germline_resource_index  = '<path to the germline_resource index>'
+      intervals                = '<path to the intervals file>'
+      known_indels             = '<path to the known_indels file>'
+      known_indels_index       = '<path to the known_indels index>'
+      mappability              = '<path to the mappability file>'
+      snpeff_db                = '<version of the snpEff DB>'
+      species                  = '<species>'
+      vep_cache_version        = '<version of the VEP cache>'
     }
     // Any number of additional genomes, key is used with --genome
   }
 }
 ```
 
-### --acLoci
-
-If you prefer, you can specify the full path to your reference genome when you run the pipeline:
+### --igenomes_base
 
-```bash
---acLoci '[path to the acLoci file]'
-```
+Specify base path to AWS iGenomes
+Default: `s3://ngi-igenomes/igenomes/`
 
-### --acLociGC
+### --igenomes_ignore
 
-If you prefer, you can specify the full path to your reference genome when you run the pipeline:
+Do not load `igenomes.config` when running the pipeline.
+You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`.
+This option will load the `genomes.config` file instead.
+You can then specify the `--genome custom` and specify any reference file on the command line or within a config file.
 
 ```bash
---acLociGC '[path to the acLociGC file]'
+--igenomes_ignore
 ```
 
-### --bwaIndex
+### --genomes_base
 
-If you prefer, you can specify the full path to your reference genome when you run the pipeline:
+Specify base path to reference genome
+
+### --save_reference
+
+Enable saving reference indexes and other files built within Sarek.
 
 ```bash
---bwaIndex '[path to the bwa indexes]'
+--save_reference
 ```
 
-### --chrDir
+### --ac_loci
 
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
 ```bash
---chrDir '[path to the Chromosomes folder]'
+--ac_loci '[path to the loci file]'
 ```
 
-### --chrLength
+### --ac_loci_gc
 
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
 ```bash
---chrLength '[path to the Chromosomes length file]'
+--ac_loci_gc '[path to the loci GC file]'
 ```
 
-### --dbsnp
+### --bwa
 
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
+> If none provided, will be generated automatically from the fasta reference.
+
 ```bash
---dbsnp '[path to the dbsnp file]'
+--bwa '[path to the BWA indexes]'
```

Comment:
This might be a bit confusing because there are multiple BWA indexes depending on which algo. you use. Maybe give more explicit example?

Reply:
For all index path examples, you should follow your </path/to/XXX> convention from the other docs

Reply:
Good point, I'll add that